<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>w_LoggingDialog</class>
 <widget class="QDialog" name="w_LoggingDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>350</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>350</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/icons/appIcon.png</normaloff>:/icons/appIcon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QLabel" name="l_tittle">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>350</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
     <pointsize>20</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Zaloguj się...</string>
   </property>
   <property name="indent">
    <number>10</number>
   </property>
  </widget>
  <widget class="Line" name="line_top">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>52</y>
     <width>350</width>
     <height>5</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_bot">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>360</y>
     <width>350</width>
     <height>5</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="l_userName">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>310</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>Nazwa użytkownika</string>
   </property>
   <property name="indent">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="f_userName">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>310</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(0, 0, 0);</string>
   </property>
   <property name="maxLength">
    <number>32</number>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="l_password">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>310</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>Hasło</string>
   </property>
   <property name="indent">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="f_password">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>310</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);</string>
   </property>
   <property name="maxLength">
    <number>32</number>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QWidget" name="w_serverConnectionArea" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>240</y>
     <width>350</width>
     <height>121</height>
    </rect>
   </property>
   <widget class="QLabel" name="l_serwer">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>250</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Serwer</string>
    </property>
    <property name="indent">
     <number>0</number>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>40</y>
      <width>320</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="b_configureButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>15</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);
border-width: 0px;
border-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/icons/configure2.png</normaloff>:/icons/configure2.png</iconset>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonFollowStyle</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="b_showHideButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>15</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);
border-width: 0px;
border-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/icons/arrow_down.png</normaloff>:/icons/arrow_down.png</iconset>
    </property>
   </widget>
   <widget class="QWidget" name="w_serverInfoArea" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>310</width>
      <height>60</height>
     </rect>
    </property>
    <widget class="QLabel" name="l_serverAddress">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>5</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Tahoma</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Adres</string>
     </property>
     <property name="indent">
      <number>0</number>
     </property>
    </widget>
    <widget class="QLabel" name="l_serverPort">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>30</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Tahoma</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Port</string>
     </property>
     <property name="indent">
      <number>0</number>
     </property>
    </widget>
    <widget class="QLabel" name="l_Address">
     <property name="geometry">
      <rect>
       <x>85</x>
       <y>5</y>
       <width>220</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Tahoma</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="indent">
      <number>0</number>
     </property>
    </widget>
    <widget class="QLabel" name="l_Port">
     <property name="geometry">
      <rect>
       <x>85</x>
       <y>30</y>
       <width>220</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Tahoma</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="indent">
      <number>0</number>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="b_logIn">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>360</y>
     <width>350</width>
     <height>40</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Zaloguj</string>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QWidget" name="w_loggingMask" native="true">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>350</width>
     <height>400</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>350</width>
     <height>400</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>350</width>
     <height>400</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 200);</string>
   </property>
   <widget class="QLabel" name="l_logInStateText">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>350</width>
      <height>400</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>WaitCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Logowanie...</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
    <property name="indent">
     <number>0</number>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
